client/packages/common/src/utils/types/TypeUtils.ts

```diff
@@ -23,3 +23,11 @@ export const getErrorMessage = (error: unknown): string => {
 };
 
 export type ArrayElement<T> = T extends (infer U)[] ? U : T;
+
+export type JsonValue =
```

Comment:
I couldn't find this type, needed it for query graphql hook. This type was actually generated by backend, I copied it over

Reply:
Also added here: https://github.com/msupply-foundation/open-msupply/issues/2951#issuecomment-2650673745